README.md

```diff
@@ -1,2 +1,133 @@
-# vc-js
-Verifiable Claims JavaScript library
+# Verifiable Credentials JS Library _(vc-js)_
+
+[![Build Status](https://travis-ci.org/digitalbazaar/vc-js.png?branch=master)](https://travis-ci.org/digitalbazaar/vc-js)
+
+> A Javascript library for issuing and verifying Verifiable Credentials
+
+## Table of Contents
+
+- [Security](#security)
+- [Background](#background)
+- [Install](#install)
+- [Usage](#usage)
+- [API](#api)
+- [Testing](#testing)
+- [Contribute](#contribute)
+- [Commercial Support](#commercial-support)
+- [License](#license)
+
+## Security
+
+TBD
+
+## Background
+
+TBD
+
+See also (related specs):
+
+* [Verifiable Credentials Data Model](https://w3c.github.io/vc-data-model/)
+* [Linked Data Signatures 1.0](https://w3c-dvcg.github.io/ld-signatures/)
+
+## Install
+
+- Node.js 8.3+ required.
+- Node.js 10.12.0+ is highly recommended due to RSA key generation speed.
+
+To install locally (for development):
+
+```
+git clone https://github.com/digitalbazaar/vc-js.git
+cd vc-js
+npm install
+```
+
+## Usage
+
+Use on the command line, or see the API section below.
+
+### Issuing Credentials
+
+To issue a Verifiable Credential, you need the following:
+
+* A URL for the public key
+* A local PEM encoded private key file
+* The input credential (`cred.json` in the example)
+
+```bash
+./bin/vc-js issue --issuer "https://example.com/keys/1" --private-key example.pem < cred.json
+```
```

Comment:
I'd prefer the command line stuff to be in a different package like `vc-js-cli` or similar. We can split this out later in the interest of time if necessary... if that's the case, please file an issue to do that later.

Reply:
No prob. Though I'm curious - any particular reason why?

Reply:
Keeps things clean, no need to pull in unnecessary dependencies, so on. Especially when this library is used in the browser, for example -- there's no "command line" there. I'd rather split the CLI stuff out and install it or not ... then have complicated configuration to try and omit it from the browser built, etc.